src/sequences/func.py

```diff
@@ -301,3 +301,25 @@ def is_vowel(char: str) -> bool:
     vowels = "aeiouAEIOU"
 
     return char in vowels
+def remove_duplicate_lists(input_list):
```

Comment:
Please follow the PEP-8 guidelines and add two empty lines before function definition.